.vscode/tasks.json

```
@@ -0,0 +1,29 @@
+{
+	"version": "2.0.0",
+	"tasks": [{
+		"label": "Start dev container",
+		"type": "shell",
+		"command": "docker compose --file .devcontainer/docker-compose.yaml up -d",
+		"windows": {
+		  "command": "docker compose --file .devcontainer\\docker-compose.yaml up -d"
```

Maybe we should not open as daemon and let VSCode handle the opened docker handle? This would ensure that the logs are visible in VSCode's interface, and also I think it would automatically close the docker stack when closing the project.
```suggestion
		"command": "docker compose --file .devcontainer/docker-compose.yaml up",
		"windows": {
		  "command": "docker compose --file .devcontainer\\docker-compose.yaml up"
```